resources/assets/less/components/gistlog.less

```diff
@@ -28,6 +32,38 @@
     }
 }
 
+.gistlog-article {
+    margin-bottom: 4em;
+}
+
+.gistlog-article__title {
+
```

Comment:
I'll clean this up, but for future, let's alphabetize CSS property names and clean up indentation in spots like this. Thanks!